snap/README.md

```diff
@@ -52,48 +58,28 @@ $ sudo cp /snap/kata-containers/current/usr/share/defaults/kata-containers/confi
 $ $EDITOR /etc/kata-containers/configuration.toml
 ```
 
-## Integration with docker ##
-
-the path to the runtime provided by the Kata Containers snap image is
-`/snap/kata-containers/current/usr/bin/kata-runtime`, this runtime must be added to
-[dockerd][5] via `systemd` or `dockerd` configuration file.
+## Integration with docker and kubernetes
 
-`/etc/systemd/system/docker.service.d/runtime.conf`
-
-```ini
-[Service]
-ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/snap/kata-containers/current/usr/bin/kata-runtime --default-runtime=kata-runtime
-```
-
-or
-
-`/etc/docker/daemon.json`
-
-```json
-{
-	"default-runtime": "kata-runtime",
-	"runtimes": {
-		"kata-runtime": {
-			"path": "/snap/kata-containers/current/usr/bin/kata-runtime"
-		}
-	}
-}
-```
-
-after having added the new runtime, the service must be reloaded and restarted
-
-```
-$ sudo systemctl daemon-reload
-$ sudo systemctl restart docker
-```
+The path to the runtime provided by the Kata Containers snap image is
+`/snap/kata-containers/current/usr/bin/kata-runtime`, it should be used to
```

Comment:
Suggested rewrite to break up the sentence and stay active: 
`/snap/kata-containers/current/usr/bin/kata-runtime`. You should use it to